apps/service_persist/lib/persist/table_manager.ex

```diff
@@ -22,6 +23,10 @@ defmodule Persist.TableManager do
   def copy(from_table, to_table) do
     impl().copy(from_table, to_table)
   end
+
+  def delete(table) do
```

Comment:
do these need `@spec` or `@impl` annotations since they're public functions?

Reply:
Maybe and I'm just not aware. I don't see def/impl on any of the sibling functions in this module, and dialyzer passes. 🤷‍♂